source/fe/fe_rannacher_turek.cc

```diff
@@ -0,0 +1,119 @@
+#include <deal.II/fe/fe_rannacher_turek.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <algorithm>
+
+#include <sstream>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+
+template <int dim>
+FE_RannacherTurek<dim>::FE_RannacherTurek(const unsigned int degree,
+                                          const unsigned int n_face_support_points) :
+  FE_Poly<PolynomialsRannacherTurek<dim>, dim>(
+    PolynomialsRannacherTurek<dim>(),
+    FiniteElementData<dim>(this->get_dpo_vector(),
+                           1,
+                           2,
+                           FiniteElementData<dim>::L2),
```

Comment:
It is scalar valued so I think H_div conforming would be confusing anyway but even if we use a system with Rannacher-Turek components the property that jumps across faces are mean value free is not enough for H_div conformity (AFAIK you need continuity of normal moments w.r.t. all trace functions, not just the constant ones)